[2024-06-18T16:05:01.103+0545] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T16:05:01.127+0545] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-18T10:10:00+00:00 [queued]>
[2024-06-18T16:05:01.134+0545] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-18T10:10:00+00:00 [queued]>
[2024-06-18T16:05:01.135+0545] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-18T16:05:01.211+0545] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-18 10:10:00+00:00
[2024-06-18T16:05:01.218+0545] {standard_task_runner.py:63} INFO - Started process 23233 to run task
[2024-06-18T16:05:01.228+0545] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-18T10:10:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/basic_dag.py', '--cfg-path', '/tmp/tmp76m4kb7e']
[2024-06-18T16:05:01.232+0545] {standard_task_runner.py:91} INFO - Job 92: Subtask backup_task
[2024-06-18T16:05:01.375+0545] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-18T10:10:00+00:00 [running]> on host manisha-Inspiron-3580
[2024-06-18T16:05:01.633+0545] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='manisha' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T10:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T10:10:00+00:00'
[2024-06-18T16:05:01.634+0545] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T16:05:01.648+0545] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-18T16:05:01.648+0545] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/manisha/airflow/bashScripts/backup.sh ']
[2024-06-18T16:05:01.653+0545] {subprocess.py:86} INFO - Output:
[2024-06-18T16:05:01.658+0545] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-18T16:05:01.662+0545] {subprocess.py:93} INFO - mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
[2024-06-18T16:05:01.856+0545] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-18T16:05:01.857+0545] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T16:05:01.985+0545] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-18T10:10:00+00:00, execution_date=20240618T101000, start_date=20240618T102001, end_date=20240618T102001
[2024-06-18T16:05:02.081+0545] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T16:05:02.137+0545] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-18T16:05:02.204+0545] {local_task_job_runner.py:222} INFO - ::endgroup::
